app/pods/components/web-components/au-dropdown/item/template.hbs

```diff
@@ -1,5 +1,11 @@
 <li class="{{this.class}}">
-  <LinkTo class="auk-menu__item active {{this.skin}}" @route={{@route}} ...attributes>
```

Comment:
Voor de leesbaarheid is `@href` misschien wel nuttig, maar eigenlijk komt `href` (zonder de @) gewoon mee over via `...attributes` :-)

Reply:
Enkele maanden terug hebben we dit met Johan Ronse besproken.
We hebben liever parameters met @ als we die meegeven aan de components ipv attributes.